src/main/java/io/cdap/e2e/utils/GcpClient.java

```diff
@@ -42,17 +44,18 @@ public static int countBqQuery(String table) throws IOException, InterruptedExce
     String projectId = SeleniumHelper.readParameters(ConstantsUtil.PROJECT_ID);
     String datasetName = SeleniumHelper.readParameters(ConstantsUtil.DATASET);
     String selectQuery = "SELECT count(*) " + " FROM `" + projectId + "." + datasetName + "." + table + "`";
-    return executeQuery(selectQuery);
+    return executeSelectQuery(selectQuery).map(Integer::parseInt).orElse(0);
   }
 
   //Deleting the table
   public static void dropBqQuery(String table) throws IOException, InterruptedException {
     String projectId = SeleniumHelper.readParameters(ConstantsUtil.PROJECT_ID);
     String datasetName = SeleniumHelper.readParameters(ConstantsUtil.DATASET);
     String dropQuery = "DROP TABLE `" + projectId + "." + datasetName + "." + table + "`";
-    executeQuery(dropQuery);
+    executeSelectQuery(dropQuery);
```

Comment:
It looks weird to call `executeSelectQuery` for drop. Please consider some other name

Reply:
changed name to executeQueryOnBQ